templates/_macros/tiered-list.jinja

```diff
@@ -22,7 +18,7 @@
     <hr class="u-hide--small">
 
     {% if has_description == true -%}
-      {%- if is_description_full_width_on_desktop == true -%}
+      {%- if variant in [2, 4] -%}
```

Comment:
Relying on integers also makes our DX here a little worse, as the developer has to check the documentation table to translate these integers to their meaning. However, we already do this for proper CSS class usage, so it's not necessarily a deal-breaker.